ingestion_tools/scripts/enqueue_runs.py

```diff
@@ -172,8 +172,9 @@ def get_datasets(
     filter_datasets = [re.compile(pattern) for pattern in filter_datasets]
     exclude_datasets = [re.compile(pattern) for pattern in exclude_dataset]
     s3_config = Config(signature_version=UNSIGNED) if anonymous else None
-    s3_client = boto3.client("s3", config=s3_config)
-    config = DBImportConfig(s3_client, None, s3_bucket, https_prefix)
+    session = Session()
+    s3_client = session.client("s3", config=s3_config)
+    config = DBImportConfig(s3_client, None, s3_bucket, https_prefix, session)
```

Comment:
the `session` argument here is supposed to be a DB session, not an aws session - can we pass in None or refactor the code so that we can instantiate this without a db session?